diff --git a/node_modules/react-native-css-interop/dist/metro/index.js b/node_modules/react-native-css-interop/dist/metro/index.js
index 8a0c01e..646461f 100644
--- a/node_modules/react-native-css-interop/dist/metro/index.js
+++ b/node_modules/react-native-css-interop/dist/metro/index.js
@@ -17,7 +17,7 @@ let haste;
 let fileSystem;
 let virtualModulesPossible = undefined;
 const virtualModules = new Map();
-const outputDirectory = path_1.default.resolve(__dirname, "../../.cache");
+const outputDirectory = path_1.default.resolve(process.cwd(), ".cache", "nativewind-css");
 const isRadonIDE = "REACT_NATIVE_IDE_LIB_PATH" in process.env;
 function withCssInterop(config, options) {
     return typeof config === "function"
diff --git a/node_modules/react-native-css-interop/dist/metro/transformer.js b/node_modules/react-native-css-interop/dist/metro/transformer.js
index 2f475cd..dcd957e 100644
--- a/node_modules/react-native-css-interop/dist/metro/transformer.js
+++ b/node_modules/react-native-css-interop/dist/metro/transformer.js
@@ -10,7 +10,8 @@ async function transform(config, projectRoot, filename, data, options) {
     const transform = config.cssInterop_transformerPath
         ? require(config.cssInterop_transformerPath).transform
         : metro_transform_worker_1.default.transform;
-    if (path_1.default.dirname(filename) !== config.cssInterop_outputDirectory ||
+    const _normalize = (p) => path_1.default.resolve(p || '').replace(/\\/g, '/').toLowerCase();
+    if (_normalize(path_1.default.dirname(filename)) !== _normalize(config.cssInterop_outputDirectory) ||
         filename.endsWith(".css")) {
         return transform(config, projectRoot, filename, data, options);
     }
